rustls-platform-verifier/Cargo.toml

```diff
@@ -47,6 +47,8 @@ android_logger = { version = "0.13", optional = true } # Only used during testin
 
 [target.'cfg(target_arch = "wasm32")'.dependencies]
 webpki-roots = "0.26"
+webpki = { package = "rustls-webpki", version = "0.102", default-features = false }
+ring = { version = "0.17.7", features = ["wasm32_unknown_unknown_js"] }
```

Comment:
I'm not sure this is correct. We just recently removed an issue where this crate unconditionally depended on _ring_ regardless of `rustls`'s set cryptography provider (https://github.com/rustls/rustls-platform-verifier/issues/102).

May I ask what led to needing to enabling this feature in production builds?

Reply:
Hey @complexspaces ,

My apologies for the late response, I was inactive over the weekend.

If I remember, adding it as a conditional dev-dependency would appease `clippy-ci`, but not `clippy-msrv-ci`.

But you're right, it was a very poor choice to promote it to a production dependency 😓